### PR TITLE
Add custom emojis to the emoji picker

### PR DESCRIPTION
**Pull request by Gargron**
image

### CODE REVIEW

File: app/javascript/mastodon/features/compose/components/emoji_picker_dropdown.js [hunk 1]
```diff
@@ -212,6 +218,27 @@ class EmojiPickerMenu extends React.PureComponent {
...
+        name,
+        short_names: [name],
+        text: '',
+        emoticons: [],
+        keywords: [name],
+        imageUrl: url,
+      });
+    });
+
+    return emojis;
```
Comment (ykzts) [inline app/javascript/mastodon/features/compose/components/emoji_picker_dropdown.js]:
```javascript
return this.props.custom_emojis.map(emoji => {
  const shortcode = emoji.get('shortcode');
  const url = emoji.get('url');
  const name = shortcode.replace(':', '');

  return {
    name,
    short_names: [name],
    text: '',
    emoticons: [],
    keywords: [name],
    imageUrl: url,
  };
});
```

How about using Array.prototype.map as above?

**Comment by ykzts** [comment]
I think that `background-repeact: no-repeat` is necessary.

File: app/javascript/mastodon/features/compose/components/emoji_picker_dropdown.js (hunk 1 above)
Comment (Gargron) [reply app/javascript/mastodon/features/compose/components/emoji_picker_dropdown.js]: I could just use `.toJS()` on the Immutable.List, probably, likely without any performance losses. But it's not a big deal...

Comment (ykzts) [reply app/javascript/mastodon/features/compose/components/emoji_picker_dropdown.js]: i agree

**Review by ykzts — APPROVED** [review]

**Pull request merged**